Solutions/TenableIO/Data Connectors/TenableIO.json

```diff
@@ -0,0 +1,156 @@
+{
+	"id": "TenableIOAPI",
+	"title": "Tenable.io Vulnerability Management",
+	"publisher": "Tenable",
+	"descriptionMarkdown": "The [Tenable.io](https://www.tenable.com/products/tenable-io) data connector provides the capability to ingest Asset and Vulnerability data into Azure Sentinel through the REST API from the Tenable.io platform (Managed in the cloud). Refer to [API documentation](https://developer.tenable.com/reference) for more information. The connector provides the ability to get data which helps to examine potential security risks, get insight into your computing assets, diagnose configuration problems and more",
+	"additionalRequirementBanner": "These queries and workbooks are dependent on a [**Tenable.io parser**](https://aka.ms/sentinel-tenableio-parser) based on Kusto to work as expected which is deployed with the Azure Sentinel Solution.",
```

Comment:
There are 2 parsers in this connector. Which parser do you want to use here?
If you want to use TenableIOVulnerabilities then use https://aka.ms/sentinel-TenableIO-TenableIOVulnerabilities-parser
and if you want to use TenableIOAssets then use https://aka.ms/sentinel-TenableIO-TenableIOAssets-parser

Reply:
I included both in this latest commit. @preetikr mentioned that having two parers was possible when we discussed. Let me know if I can only pick one instead, and I will change to just the vulnerability parser.